go/app/main.go

```diff
@@ -20,22 +24,131 @@ type Response struct {
 	Message string `json:"message"`
 }
 
+type ResponseItems struct {
+	Items []ItemInfo `json:"items"`
+}
+
+type ItemInfo struct {
+	Name          string `json:"name"`
+	Category      string `json:"category"`
+	ImageFilename string `json:"image_filename"`
+}
+
 func root(c echo.Context) error {
 	res := Response{Message: "Hello, world!"}
 	return c.JSON(http.StatusOK, res)
 }
 
 func addItem(c echo.Context) error {
+	// Open and read items.json
+	var jsonItems ResponseItems
+	jsonFile, err := os.Open("items.json")
```

Comment:
`os.Open("items.json")` operates with relative path. It means that the result of this execution will change depending on the directory you run your golang application from (my first run failed because I run the application from the `/go` directory😅)
It won't matter in our case because in future we will wrap the application in the docker container and get rid of the file, so no problem, just sharing fun fact😅